src/tools/rbd/action/Bench.cc

```diff
@@ -316,7 +364,8 @@ void add_bench_common_options(po::options_description *positional,
     ("io-size", po::value<Size>(), "IO size (in B/K/M/G/T) [default: 4K]")
     ("io-threads", po::value<uint32_t>(), "ios in flight [default: 16]")
     ("io-total", po::value<Size>(), "total size for IO (in B/K/M/G/T) [default: 1G]")
-    ("io-pattern", po::value<IOPattern>(), "IO pattern (rand or seq) [default: seq]");
+    ("io-pattern", po::value<IOPattern>(), "IO pattern (rand or seq) [default: seq]")
+    ("rw-proportion", po::value<uint64_t>(), "read proportion in rw (<= 100) [default: 50]");
```

Comment:
```nw-proportion``` name doesn't really tell me which percentage will be used for reads vs writes in a mixed workload. Perhaps ```rw-mix-read```?